test/index/namespaces/namespaces.snapshot.cc

```diff
@@ -65,3 +65,26 @@
   IDENTITY(namespace in_macro { })
 //^^^^^^^^ definition [..] in_macro/
 //^^^^^^^^ reference [..] namespaces.cc:45:9#
+  
+  namespace a {
+//          ^ definition [..] a/
+    namespace c {
+//            ^ definition [..] a/c/
+      enum E { E0 };
+//         ^ definition [..] a/c/E#
+//             ^^ definition [..] a/c/E0.
```

Comment:
> This is legal and x is a bar, but won't (1) and (2) both be called foo?

There won't be a collision since they have separate suffixes. `#` is for types and `.` is for values. https://github.com/sourcegraph/scip/blob/main/scip.proto#L113-L114